install/update_94_95.php

```diff
@@ -257,6 +257,80 @@ function update94to95() {
          ['name' => 'tickettemplate']
       )
    );
+
+   //create template tables for other itil objects
+   foreach(['change', 'problem'] as $itiltype) {
+      if (!$DB->tableExists('glpi_{$itiltype}templates')) {
+         $query = "CREATE TABLE `glpi_{$itiltype}templates` (
+            `id` int(11) NOT NULL AUTO_INCREMENT,
+            `name` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+            `entities_id` int(11) NOT NULL DEFAULT '0',
+            `is_recursive` tinyint(1) NOT NULL DEFAULT '0',
+            `comment` text COLLATE utf8_unicode_ci,
+            PRIMARY KEY (`id`),
+            KEY `name` (`name`),
+            KEY `entities_id` (`entities_id`),
+            KEY `is_recursive` (`is_recursive`)
+            ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;";
+         $DB->queryOrDie($query, "add table glpi_{$itiltype}templates");
+         $migration->addPostQuery(
+            $DB->buildInsert(
+               "glpi_{$itiltype}templates",
+               [
+                  'id'           => 1,
+                  'name'         => 'Default',
+                  'is_recursive' => 1
+               ]
+            )
+         );
+      }
+
+      if (!$DB->tableExists("glpi_{$itiltype}templatehiddenfields")) {
+         $query = "CREATE TABLE `glpi_{$itiltype}templatehiddenfields` (
+            `id` int(11) NOT NULL AUTO_INCREMENT,
+            `{$itiltype}templates_id` int(11) NOT NULL DEFAULT '0',
+            `num` int(11) NOT NULL DEFAULT '0',
+            PRIMARY KEY (`id`),
+            UNIQUE KEY `unicity` (`{$itiltype}templates_id`,`num`)
```

Comment:
Other `*template*fields` have this key.
```suggestion
            UNIQUE KEY `unicity` (`{$itiltype}templates_id`,`num`),
            KEY `{$itiltype}templates_id` (`{$itiltype}templates_id`)
```